packages/lucide-react/src/createLucideIcon.ts

```diff
@@ -39,16 +50,14 @@ const createLucideIcon = (iconName: string, iconNode: IconNode): LucideIcon => {
         },
         [
           ...iconNode.map(([tag, attrs]) => createElement(tag, attrs)),
-          ...(
-            (Array.isArray(children) ? children : [children]) || []
-          )
-        ],
-      ),
+          ...((Array.isArray(children) ? children : [children]) || []),
```

Comment:
`Array.isArray(children) ? children : [children]` is always `truthy`.

```suggestion
          ...(Array.isArray(children) ? children : [children]),
```